authors/apps/authentication/views.py

```diff
@@ -6,16 +12,44 @@
 from .renderers import UserJSONRenderer
 from . import serializers
 from .serializers import (
-    LoginSerializer, RegistrationSerializer, UserSerializer
+    LoginSerializer, RegistrationSerializer, UserSerializer,
+    ResetSerializerEmail, ResetSerializerPassword
 )
-
+from authors.apps.core.permissions import IsAuthorOrReadOnly
+from .models import User
 
 class RegistrationAPIView(generics.CreateAPIView):
     # Allow any user (authenticated or not) to hit this endpoint.
     permission_classes = (AllowAny,)
     renderer_classes = (UserJSONRenderer,)
     serializer_class = RegistrationSerializer
 
+    #Method to Send emails
+    def send_link(self,email):
+        payload = {'email':  email,"iat": datetime.now(),
+                    "exp": datetime.utcnow() + timedelta(minutes=5)}
```

Comment:
Excess space after email key, missing whitespace after `email,`, put each key on its own line to aid readability